pom.xml

```diff
@@ -16,8 +16,8 @@
     <revision>1.0</revision>
     <changelist>-SNAPSHOT</changelist>
     <gitHubRepo>jenkinsci/${project.artifactId}-plugin</gitHubRepo>
-    <jenkins.version>2.222.4</jenkins.version>
```

Comment:
We intentionally depend on older version of Jenkins in order to maintain compatible with a wide range of Jenkins. There's nothing in this plugin that requires new version of Jenkins, so it's important to stick to the current base Jenkins version.